CODEOWNERS

```diff
@@ -16,6 +16,7 @@
 /packages/plugins/*_openscap/ @theforeman/packaging @theforeman/openscap
 /packages/foreman/rubygem-ldap_fluff/ @theforeman/packaging @theforeman/ldap-fluff
 /packages/plugins/*_webhooks/ @theforeman/packaging @theforeman/webhooks
+/packages/plugins/*_smart_proxy_shellhooks/ @theforeman/packaging @theforeman/webhooks
```

Comment:
The underscore at the beginning won't match. Given how specific it is, perhaps write it out?

Reply:
/me sighs

https://github.com/theforeman/foreman-packaging/pull/11813